x/oracle/types/types.pb.go

```diff
@@ -27,24 +27,28 @@ const _ = proto.GoGoProtoPackageIsVersion3 // please upgrade the proto package
 type StatusText int32
 
 const (
-	StatusText_UNKNOWN_STATUS_TEXT StatusText = 0
-	StatusText_PEDNING_STATUS_TEXT StatusText = 1
-	StatusText_SUCCESS_STATUS_TEXT StatusText = 2
-	StatusText_FAILED_STATUS_TEXT  StatusText = 3
+	// Default value
+	StatusText_STATUS_TEXT_UNSPECIFIED StatusText = 0
+	// Pending status
+	StatusText_STATUS_TEXT_PEDNING StatusText = 1
```

Comment:
Still got a type in the oracle proto PEDNING

Reply:
oh whoops thanks.